### PR TITLE
Groomer Ratings Feature

### PR DESCRIPTION
**Pull request by davisblakep**
Added empty rating stars to groomers in the search results and disabled option to click them so people aren't able to publicly rate if not signed in. Waiting to connect rating with backend. Added small cosmetic fixes on the site for various stylings.

## Description ##

DESCRIBE YOUR PULL REQUEST'S CHANGES AND REFERENCE THE FEATURE/USERSTORY/BUG IT'S REALATED TO.  REPLACE THIS TEXT.

## Type ##

- [ ] This is a bugfix.
- [x] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [ ] Have you linked this pull request to a Trello card?
- [x] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing 

## Review ##

- [x] Is this a work in progress?
- [x] Is this ready to be reviewed?
- [x] Have you added two reviewers to this pull request?

### CODE REVIEW

File: src/components/search/SearchResults/SearchResultsCard.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import 'antd/dist/antd.css';
 import { Card } from 'antd';
+import { Rate } from 'antd';
```
Comment (FIOpwK) [inline src/components/search/SearchResults/SearchResultsCard.js:5]: Do these antd components always import this way, or is grouping them as {Card, Rate} not readable?

**Review by FIOpwK — DISMISSED** [review]

**Review by LandonDvong — DISMISSED** [review]
I'm guessing the Rating BE is still in progress, this is still a good foundation for our rating system none the less. Squeaky clean work!

File: src/components/search/SearchResults/SearchResultsCard.js (hunk 1 above)
Comment (davisblakep) [reply src/components/search/SearchResults/SearchResultsCard.js:5]: These can also be grouped like: import { Card, Rate } from 'antd';

**Review by jarrod847 — DISMISSED** [review]

**Review by jarrod847 — APPROVED** [review]

**Review by linzyk86 — APPROVED** [review]
Looks good

**Pull request merged**